docs/src/modules/components/DiamondSponsors.js

```diff
@@ -129,16 +119,51 @@ export default function DiamondSponsors() {
               })
             }
           />
-        </a>
+        </StyledAnchor>
         <Link
           aria-label={t('diamondSponsors')}
           rel="noopener noreferrer"
           href="/material-ui/discover-more/backers/#diamond"
-          fontSize={14}
-          color="text.secondary"
-          // eslint-disable-next-line material-ui/no-hardcoded-labels
+          sx={(theme) => ({
+            width: '100%',
+            p: 1.5,
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            justifyContent: 'center',
+            borderRadius: 1,
+            border: '1px dashed',
+            transition: theme.transitions.create(['color', 'border-color', 'background-color']),
+            backgroundColor: alpha(theme.palette.primary[50], 0.5),
+            borderColor: (theme.vars || theme).palette.primary[200],
+            boxShadow: `inset 0 1px 1px ${
+              (theme.vars || theme).palette.grey[50]
+            }, 0 1px 2px ${alpha(theme.palette.primary[100], 0.8)}`,
+            '&:hover': {
+              backgroundColor: alpha(theme.palette.primary[100], 0.5),
+              borderColor: (theme.vars || theme).palette.primary[300],
+            },
+            ...theme.applyDarkStyles({
+              backgroundColor: alpha(theme.palette.primary[400], 0.05),
+              borderColor: alpha(theme.palette.primary[300], 0.3),
+              boxShadow: `inset 0 1px 1px ${
+                (theme.vars || theme).palette.primaryDark[800]
+              }, 0 1px 0.5px ${(theme.vars || theme).palette.common.black}`,
+              '&:hover': {
+                backgroundColor: alpha(theme.palette.primary[400], 0.1),
+                borderColor: (theme.vars || theme).palette.primary[400],
+              },
+            }),
+          })}
         >
-          +1
+          <Typography variant="caption" fontWeight="semiBold" textAlign="center">
+            {/* eslint-disable-next-line material-ui/no-hardcoded-labels */}
+            {'Become a diamond sponsor!'}
+          </Typography>
+          <Typography variant="caption" fontWeight="regular" color="text.secondary">
+            {/* eslint-disable-next-line material-ui/no-hardcoded-labels */}
+            {'1/3 spots left'}
```

Comment:
@samuelsycamore just checking if this reads clearly as "1 out of 3 spots left" — not sure; what do you think?

Reply:
hmm, maybe "one spot left" is clearer?

Reply:
"One" or "1"? 😬

Reply:
What about "Last slot available" or something like this?